includes/class-freemius.php

```diff
@@ -20980,7 +20980,7 @@ private function _sync_plugin_license(
                         $this->switch_to_blog( $this->_storage->network_install_blog_id );
                     }
 
-                    // Show API messages only if not background sync or if paying customer.
+	                // Show API message only if not background sync or if paying customer. And if it's a background process, show message only if it's a paying customer.

```

Comment:
@msladu The extra clarification you have added is redundant, please remove it.

Reply:
@vovafeldman Ok. Removing it.